### PR TITLE
Change where to get latest .gitignore from

### PR DESCRIPTION
**Pull request by danielchalmers**
What do you think about this change?

The original VisualStudio.gitignore hasn't been updated in over a year, but this one is updated more regularly.

A URL could be easier but I think it makes sense that if you got it from `dotnet` in the first place, you'd be able to update it from there too.

### CODE REVIEW

File: template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Ignore Visual Studio temporary files, build results, and
 ## files generated by popular Visual Studio add-ons.
 ##
-## Get latest from https://github.com/github/gitignore/blob/main/VisualStudio.gitignore
+## Get latest from `dotnet new gitignore`
```
Comment (vlada-shubina) [inline template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.gitignore:4]: @baronfel do you have thoughts about it?

Comment (baronfel) [reply template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.gitignore:4]: Ideally they all would be the same source, but I'm comfortable leaning on `dotnet new` here. This does put more pressure on us to do something with https://github.com/dotnet/templating/issues/4126, but I think the net result is better for our users.

**Comment by YuliiaKovalova** [comment]
@GangWang01 , could you fix the failed tests?
then we can merge this PR

**Comment by GangWang01** [comment]
> @GangWang01 , could you fix the failed tests? then we can merge this PR
I don't have permission to edit PR. So I created another PR #32283.

**Comment by YuliiaKovalova** [comment]
the change was merged in https://github.com/dotnet/sdk/pull/32283.

**Pull request closed without merging**